build.gradle

```diff
@@ -110,11 +110,34 @@ ext {
 }
 
 subprojects { project ->
+    task pullInstrumentedTestAmplifyConfig() {
```

Comment:
Can you add these new task definitions (including `runCodeBuildScript`) under `configuration/instrumentation-tests.gradle` or something?